src/graphics/device.js

```diff
@@ -1839,6 +1858,46 @@ pc.extend(pc, function () {
 
         /**
          * @function
+         * @name pc.GraphicsDevice#setTransformFeedbackBuffer
```

Comment:
Doesn't justifies the way device methods are exposed for not reason, polluting API Reference with dozen of methods :)